buildkit/packaging/opensuse.py

```diff
@@ -6,11 +6,12 @@
 
```

Comment:
If you haven't already, please check and fix warnings and errors detected by `developer_utilities/pylint_buildkit.py` for this code if you haven't already. It will be much appreciated :+1: